app/routers/dogfinder.py

```diff
@@ -174,26 +188,30 @@ async def add_documents(documentRequest: DocumentRequest):
         # Create the embedding model
         logger.info(f"Creating embedding model")
         embedding_model, cache_info = create_embedding_model()
-            
+
         # Embed the documents images
-        embedding_results = embed_documents([create_pil_image(document.image) for document in documentRequest.documents], embedding_model)
+        embedding_results = embed_documents(
+            [create_pil_image(document.image) for document in documentRequest.documents], embedding_model)
 
         # Iterate over the documents and add them to the database
         for i, document in enumerate(documentRequest.documents):
             try:
                 # Create the data object
-                data_properties = create_data_properties(document)       
-         
+                data_properties = create_data_properties(document)
+
                 # Create a uuid based on the filename
-                data_properties["uuid5"] = generate_uuid5({ "filename": document.filename })
```

Comment:
Same comment as above with the uuid5. no need for this.

Reply:
same response, it is used in the insert